FindMyCoffee.playground/Sources/FindMyCoffeTests/CoffeeShopLocationsParserTests.swift

```diff
@@ -0,0 +1,21 @@
+import XCTest
+
+
+public class CoffeeShopLocationsParserTests: XCTestCase {
+    
+    var coffeeShops:[CoffeeShop]?
+    
+    public override func setUp() {
+        super.setUp()
+        let input = "Starbucks Seattle,47.5809,-122.3160"
+        coffeeShops = try? CoffeeShopLocationsParser.parse(inputDataCSVString: input)
+    }
+    
+    public func test() {
```

Comment:
For the locations without a name, I can provide a default generic name, but if one of coordinates are wrong I can't think to a valid solution without affecting the precision of location. Also, if the input data is inconsistent I have that exception that is triggered.

Reply:
You could also skip over bad records, and still over the user a solution from the valid ones, it won't be that nice but at least they can still grab some coffee.

Reply:
Instead of throwing an error I can use continue to skip over invalid data, and a solution is still provided.  
I'm not very pleased about doing that, i'd prefer a full error instead a half of the solution, but in some cases, it may be useful.